gtsfm/bundle/bundle_adjustment.py

```diff
@@ -306,16 +306,16 @@ def evaluate(
     def create_computation_graph(
         self,
         sfm_data_graph: Delayed,
-        absolute_pose_priors: List[Delayed],
-        relative_pose_priors: Dict[Tuple[int, int], Delayed],
+        absolute_pose_priors: List[Optional[PosePrior]],
+        relative_pose_priors: Dict[Tuple[int, int], Optional[PosePrior]],
```

Comment:
Maybe we should insist the Dict has non None values. We now (since this PR) check for the key when creating the plate anyway.

Reply:
Yes I agree. We can kill the Optional in relative priors.

Reply:
Done